src/containers/Order/styles.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '@scss/mixins';
+
+.adminOrderContainer {
+  @include flex-common(space-evenly, center);
+}
+
+.mb_4 {
+  margin-bottom: 1.5rem !important;
+}
+
+.card {
+  position: relative;
+  box-shadow: 0 0.1rem 0.25rem rgb(0 0 0 / 8%);
+  display: flex;
+  flex-direction: column;
+  min-width: 0;
+  word-wrap: break-word;
+  background-color: #fff;
+  background-clip: border-box;
+  border: 0 solid rgba(222, 226, 230, 70%);
+  border-radius: 0.25rem;
+}
+
+.cardBody {
+  flex: 1 1 auto;
+  padding: 1.3rem;
+}
+
+.cardHeader:first-child {
+  padding: 1.3rem;
+  margin-bottom: 0;
+  border-bottom: 1px solid rgba(222, 226, 230, 70%);
+  border-radius: 0.25rem 0.25rem 0 0;
+}
+
+.formControl {
+  padding: 0.5rem 0.75rem;
+  font-size: 1rem;
+  font-weight: 400;
```

Comment:
This can be replaced with `@include text-style(1rem, 400)`

Reply:
clear